app/uk/gov/hmrc/gform/lookup/LookupOptions.scala

```diff
@@ -49,8 +49,9 @@ object LookupOptions {
   def getLookupValue(lookupInfo: LookupInfo, columnName: String): Option[String] =
     (lookupInfo, columnName) match {
       // format: off
-      case (CountryLookupInfo(_, _, _, _, region), CsvColumnName.region)                  => Some(region.region)
-      case (CountryLookupInfo(id, _, _, _, _), CsvColumnName.countryCode)                 => Some(id.id)
+      case (CountryLookupInfo(_, _, _, _, region, _), CsvColumnName.region)               => Some(region.region)
+      case (CountryLookupInfo(id, _, _, _, _, _), CsvColumnName.countryCode)              => Some(id.id)
+      case (CountryLookupInfo(_, _, _, _, _, columns), column)                            => columns.get(column)
```

Comment:
```suggestion
      case (CountryLookupInfo(_, _, _, _, _, columns), column)                            => Some(columns.getOrElse(column, throw new Exception(s"Invalid column name $column")))
```

Reply:
it will change the current behaviour when we just ignore an non-existent column. We even have a test for that. Is that okay?
https://github.com/hmrc/gform-frontend/blob/2c2856d47f0270b31541871b4c2b6dec04627b30/test/uk/gov/hmrc/gform/lookup/LookupOptionsSpec.scala#L88